clients/admin-ui/src/features/privacy-requests/DenyPrivacyRequestModal.tsx

```diff
@@ -44,37 +44,37 @@ const DenyPrivacyRequestModal = ({
     returnFocusOnClose={false}
   >
     <ModalOverlay />
-    <ModalContent width='100%' maxWidth='456px'>
+    <ModalContent width="100%" maxWidth="456px">
```

Comment:
@TheAndrewJackson I know you had set `jsxSingleQuote` to true, but is there any chance we can keep double quotes in JSX? I've never worked with JSX single quotes before. I don't want to be pedantic but I'm wondering what the purpose of them is? Just consistency?

Reply:
We can definitely keep double quotes. I used single because I was following someone else who used single quotes. If double quotes are preferred then let's use them instead. Especially if the `fidesctl` repo is using double quotes. 

It's not a big deal anyway. As long as the formatter automatically sets it I'm fine with either.

Reply:
Cool. Let's update the Prettier file to use double quotes then – thanks for your flexibility here!